### PR TITLE
Fix additionalTrapElements to accept HTMLelements as well

### PR DESCRIPTION
**Pull request by GretaD**
### ☑️ Resolves

* Fix https://github.com/nextcloud/mail/pull/9144#issuecomment-1900748293

### 🖼️ Screenshots

🏚️ Before | 🏡 After
---|---
B | A

### 🚧 Tasks

- [x] Test if it works on mail, it works.

### 🏁 Checklist

- [x] ⛑️ Tests are included or are not applicable
- [x] 📘 Component documentation has been extended, updated or is not applicable

### CODE REVIEW

**Review by ShGKme — APPROVED** [review]

**Comment by GretaD** [comment]
the error is gone, i can see the tool, but i cannot see the dropdown list. It is shown for a second and goes away. But im not sure if this issue with the drop down is connected to the trap element

File: src/components/NcDialog/NcDialog.vue [hunk 1]
```diff
@@ -162,7 +162,14 @@ export default defineComponent({
 		/** Additional elements to add to the focus trap */
```
Comment (susnux) [inline src/components/NcDialog/NcDialog.vue:162]: Maybe add that we accept the same as NcModal

**Review by susnux — APPROVED** [review]
Nice! But will not work. We need to pass it to `NcModal`.
This was forgotten :see_no_evil: So add `additionalTrapElements` to `modalProps`

**Comment by GretaD** [comment]
> Nice! But will not work. We need to pass it to `NcModal`. This was forgotten 🙈 So add `additionalTrapElements` to `modalProps`

ah, ok, will do. test and then will merge this one

**Comment by GretaD** [comment]
tested and works now :)

**Review by susnux — APPROVED** [review]

**Pull request merged**